xml/vt_installation.xml

```diff
@@ -275,16 +275,15 @@
   </para>
   <para>
    The firmware used by virtual machines is auto-selected. The auto-selection
-   is based on the *.json files in the
-   <package>qemu-ovmf-<replaceable>ARCH</replaceable></package> package. The
+   is based on the *.json files in the firmware package described above. The
```

Comment:
Should we use a wildcard file spec in the text like this?

Reply:
```suggestion
   is based on the JSON files in the firmware package described above. The
```